### ISSUE TITLE
Enhancement: Forum channel

### ISSUE THREAD

**Issue by JulesdeCube** [Lucky3028/terraform-provider-discord#77]
## Description

Discord introduce in August of last year from channel (see [this post](https://discord.com/community/creating-value-with-conversation)) and the last version of discordGo support it.

- discord API: https://discord.com/developers/docs/topics/threads#forums
- discordGo: https://github.com/bwmarrin/discordgo/releases/tag/v0.27.0

## Request

Can we have a resource to manage forum channel `discord_forum_channel` ?

**Comment by Lucky3028** [comment]
I can't implement forum channel so that his provider uses [disgord](https://github.com/andersfylling/disgord/tree/master) and the library doesn't implement it. Sorry.

**Comment by JulesdeCube** [comment]
Ok, Excuse me for the inconvinent.

I will keep the issus open until there is an implementation in disgord.

And also thanks you for your porvider.

**Comment by Lucky3028** [comment]
I had never heard of discordgo before, but since discordgo seems to be more popular than disgord, there is a way to switch to it. Thanks!

**Comment by Lucky3028** [comment]
Yes, I think that it's possible, but I am not very motivated to do much maintenance and development since I am not using it anymore and I cannot promise when it will be implemented. Sorry.
